src/js/actions/export.js

```diff
@@ -119,6 +120,40 @@ define(function (require, exports) {
     };
 
     /**
+     * Get current status of generator
+     * @private
+     * @return {Promise.<boolean>}
+     */
+    var _getGeneratorStatus = function () {
+        return descriptor.playObject(generatorLib.getGeneratorStatus())
+            .then(function (status) {
+                return objUtil.getPath(status, "generatorStatus.generatorStatus") === 1;
+            });
```

Comment:
Might wanna add a second function to this `then` handler to catch potential failures and return false?